### DISCUSSION TITLE
Manually download GIF option after capture complete

### DISCUSSION THREAD

**Discussion by cuongdo926** [amandaghassaei/canvas-capture#3] [Q&A]
Hi, is there any way to manually download GIF after export?

**Comment by amandaghassaei** [comment]
it's set to auto download, you want a callback to handle the blob yourself?  that is not currently supported.

**Reply by cuongdo926** [reply]
Yes, would you mind adding it?

**Reply by amandaghassaei** [reply]
ok try the latest version 1.0.12, pass in `onExport(blob, filename) => {}` callback into the gif record options and that should disable auto-save.  I've added docs in the readme.  Let me know how that works!

**Reply by cuongdo926** [reply]
Thanks for your help. But it doesn't work. I checked your code and notice that you don't pass custom hooks into the capture variable.

![image](https://user-images.githubusercontent.com/85761020/149268034-fc33b7d9-e08e-48b2-b15a-1727bc953daa.png)

**Reply by amandaghassaei** [reply]
oops good catch, just published a working version: 1.0.13.

thanks for checking that!
